cypress/integration/channel/message_spec.js

```diff
@@ -38,4 +38,27 @@ describe('Message', () => {
         // * Check profile image is not visible
         cy.get('#postListContent > .post.current--user > .post__content > .post__img').last().should('be.visible').should('be.empty');
     });
+
+    it('M14012 Focus move to main input box when a character key is selected', () => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        // 2. Left click on any post to move the focus out of the main input box
+        cy.get('#post_o55dfx8ooi8azmy83ypgor68ec').click();
```

Comment:
The postId `o55dfx8ooi8azmy83ypgor68ec` can't be hardcoded as it's different for every test instance.  I'd recommend to add an initial step of posting a message and getting post ID of such message from the `postListContent`.  Once you have such post ID, you may used it to get element IDs, like:
- `cy.get(`#post_${postId}`)`
- `cy.get(`#CENTER_dropdown_${postId} .dropdown-toggle`)` 

Let me know if you need help on how to that.

Reply:
Thank you for the feedback! I've pushed a fix. I added a couple of custom commands to try to clean things up but I'm not sure if that makes sense in this case, so just let me know and I can remove them. Also, I hit the max-nested-callbacks for eslint because cypress uses closures for variables. I increased the max just for my test but maybe you would like that done differently?

Reply:
I think I added all the improvements you suggested. Thank you for your help!